src/amo/searchUtils.js

```diff
@@ -21,9 +27,11 @@ export const paramsToFilter = {
   page_size: 'page_size',
   promoted: 'promoted',
   q: 'query',
+  ...generate_threshold_params({ param: 'ratings' }),
   sort: 'sort',
   tag: 'tag',
   type: 'addonType',
+  ...generate_threshold_params({ param: 'users' }),
 };
```

Comment:
The general idea of `paramsToFilter` is to convert API params to filters in addons-frontend and to use more meaningful names in some cases. I see that we aren't very consistent, though, but I am wondering if we shouldn't have a different pattern for the filter names.

I don't really want us to have to deal with `foo__lte`, etc. OTOH using `fooLessThanOrEqual` seems a bit too verbose.

Reply:
One thing we could do is leaving a `// TODO` above `generateThresholdParams` that says something like:

```
// TODO: we need to specify these API params in `paramsToFilter` so that the app uses them 
// when making API calls but ideally we'd revisit the filter names when we need to use the 
// filters in our code (because `foo__lte` isn't a great name).
```

Reply:
For `generateThresholdParams`, we could generate them as `fooGTE`, etc, if that's better?

Reply:
yeah, I am not sure `fooGTE` is a lot better hence my suggestion to add a comment in the meantime...